src/helpers/utils.js

```diff
@@ -0,0 +1,58 @@
+import fs from 'fs';
+import { URL } from 'url';
+
+/**
+ * Function to read a json file
+ * @param {*} filename  file name or path to read
+ * @returns the JSON data in the file
+ */
+export const readJSONFile = (filename) => {
+    let rawdata = fs.readFileSync(filename);
+    let data = JSON.parse(String(rawdata));
+    return data
+}
```

Comment:
this is for a lot of functions, you are not error handling. for e.g. in this case, it can fail cause a file is not found or it can fail cause json is not correct 

`export const readJSONFile = (filename) => {
  try {
    const rawdata = fs.readFileSync(filename);
    const data = JSON.parse(String(rawdata));
    return data;
  } catch (err) {
    console.error(`Error reading or parsing ${filename}:`, err);
    return null;
  }
}`

Reply:
Added error handling for this function